lib/semantics/expression.cc

```diff
@@ -422,21 +422,20 @@ static MaybeExpr CompleteSubscripts(
 // Applies subscripts to a data reference.
 static MaybeExpr ApplySubscripts(ExpressionAnalysisContext &context,
     DataRef &&dataRef, std::vector<Subscript> &&subscripts) {
-  return std::visit(
-      common::visitors{
-          [&](const Symbol *symbol) {
-            return CompleteSubscripts(
-                context, ArrayRef{*symbol, std::move(subscripts)});
-          },
-          [&](Component &&c) {
-            return CompleteSubscripts(
-                context, ArrayRef{std::move(c), std::move(subscripts)});
-          },
-          [&](auto &&) -> MaybeExpr {
-            CHECK(!"bad base for ArrayRef");
-            return std::nullopt;
-          },
-      },
+  return std::visit(common::visitors{
+                        [&](const Symbol *symbol) {
+                          return CompleteSubscripts(context,
+                              ArrayRef{*symbol, std::move(subscripts)});
+                        },
+                        [&](Component &&c) {
+                          return CompleteSubscripts(context,
+                              ArrayRef{std::move(c), std::move(subscripts)});
+                        },
+                        [&](auto &&) -> MaybeExpr {
+                          CHECK(!"bad base for ArrayRef");
+                          return std::nullopt;
+                        },
+                    },
```

Comment:
This might have been formatted with the wrong version of clang-format.

Reply:
ok